platformsh-flex-env.php

```diff
@@ -142,3 +145,37 @@ function mapPlatformShDatabase() : string
 
     return $dbUrl;
 }
+
+function mapPlatformShMongoDatabase(): void
+{
+    /** Set the MONGODB_SERVER, MONGODB_DB, MONGODB_USERNAME, and MONGODB_PASSWORD for Doctrine, if necessary.
+     * Usage of the full dsn string is NOT working with doctrine-odm-bundle, that's why
+     * we need those 4 env variables.
+     * 
+     * Here is a example of the related doctrine-odm-bundle:
+     * doctrine_mongodb:
+     *     connections:
+     *         default:
+     *             server: '%env(MONGODB_SERVER)%'
+     *             options: { username: '%env(MONGODB_USERNAME)%', password: '%env(MONGODB_PASSWORD)%', authSource: '%env(MONGODB_DB)%' }
+     *     default_database: '%env(MONGODB_DB)%'
+     *
+     * For more information: https://symfony.com/doc/master/bundles/DoctrineMongoDBBundle/index.html
+     */
```

Comment:
Docblock goes on top of the function in PHP.  This is where it would go in Python. :smile:

Reply:
changed!